TrainingApp/Controller/HomeViewController.swift

```diff
@@ -106,6 +107,21 @@ final class HomeViewController: UIViewController {
             task.resume()
         }
     
+    private func searchFlights(from url: String) {
+        let url = URL(string: url)
+        let token = "3PG3AfG91GK9FUwbkhQpIhIXzh2L"
+        var request = URLRequest(url: url!)
```

Comment:
Let's choose another way to unwrap an optional value because it is potential application  crash

Reply:
Fixed using 'guard let'